tests/integration/targets/module_helper/tasks/msimple_output_conflict.yml

```diff
@@ -28,4 +28,27 @@
       - simple1.abc == "abc"
       - simple1 is not changed
       - simple1.value is none
-      - 'simple2._msg == "a message in a bottle"'
+      - >
+        "_msg" not in simple2
```

Comment:
Isn't this now strictly a breaking change?

Reply:
It "is" in the sense that it breaks with the previous alleged behaviour, but that behaviour was not working until #5755 . And this is not a documented (yes, I should address that at some point) behaviour either. IMO it is extremely unlikely we are breaking any code out there, I would be willing to mark it as a breaking change in the changelog and just move on.

Reply:
One problem with this being a breaking change is that if another collection uses MH and wants to use this feature, the behavior will differ depending on which version of community.general is used.

I think there are three possibilities:
1. also backport this feature to the same versions the bugfix was ported to;
2. revert the bugfix for older versions (so it just won't work for them at all);
3. document this as a breaking change (as you suggest).

I currently tend to 3. (as 1. and 2. suck more ;) ), arguing that it is not a real "breaking change" since it will not break any already existing code. We should still put it as `breaking_change` into the changelog so that if someone wants to start using this feature, they know precisely when its behavior changed.

Reply:
Agreed, adding that

Reply:
done